pkg/virt-controller/watch/dra/dra.go

```diff
@@ -53,8 +53,8 @@ const (
 	tombstoneGetObjectErrFmt = "couldn't get object from tombstone %+v"
 
 	indexByNodeName              = "byNodeName"
-	PCIAddressDeviceAttributeKey = "resource.kubernetes.io/pcieRoot"
-	MDevUUIDDeviceAttributeKey   = "resource.kubernetes.io/mDevUUID"
+	PCIAddressDeviceAttributeKey = "resource.kubernetes.io/pciAddress"
```

Comment:
s/pciAddress/pciBusID/ ?

Reply:
ok but will need to fix here as well then
https://github.com/k8snetworkplumbingwg/dra-driver-sriov/pull/44/files#diff-6572dddc535b4d2c98746d71dde97840af1328797e193db28376939a320be8f8R43